src/App.js

```diff
@@ -118,6 +116,7 @@ class Root extends React.Component {
               <NavigationRoute exact path="/statistics" component={Dashboard} />
               <NavigationRoute exact path="/token_detail/:tokenUID" component={TokenDetail} />
               <NavigationRoute exact path="/address/:address" component={AddressDetail} />
+              <NavigationRoute exact path="/nano_contract/detail/:nc_id" component={NanoContractDetail} />
```

Comment:
Is this part of this PR?

Reply:
Yes, this belongs to the other PR (the nano integration PR), my bad.

Removed at https://github.com/HathorNetwork/hathor-explorer/pull/252/commits/9b6888b5228997e492b5bb854eb0eef51c56e8a5